app/views/employees/index.html.slim

```diff
@@ -0,0 +1,38 @@
+.container
+  .page-header
+    h1 Employees
+
+  .float-left
+    table
+      thead
+        tr
+          th No.
+          th First Name
+          th Last Name
+          th Department
+          th
+
+      tbody
+        - @employees.each do |employee|
+          tr
+            td = employee.id
+            td = employee.first_name
+            td = employee.last_name
+            td = employee.department
+            td
+              = link_to 'view', employee_path(employee.id), class: 'btn btn-primary'
+              = link_to 'Edit', edit_employee_path(employee), class: 'btn btn-primary'
+              = link_to 'Delete', employee, data: {:confirm=>'Are you sure?'}, :method => :delete, class: 'btn btn-primary'
```

Comment:
It's nice to have those other spaces, it was just the space off the back that needed to come off.
{ :confirm => 'Are you sure?' },